AEPServices/Sources/ui/UIService.swift

```diff
@@ -16,21 +16,35 @@ import UIKit
 ///
 /// UIService for creating UI elements
 ///
+@objc
```

Comment:
I noticed that ServiceProvider is not exposed to Objective-C. Any use case on why we need to expose this class? If so, we should prefix it with `AEP`

Reply:
`UIService` is a protocol.

i normally wouldn’t hesitate to add an `AEP` prefix to it, except we already have a class named `AEPUIService` (which implements `UIService`).

thoughts?